CONTRIBUTING.md

```diff
@@ -526,6 +616,150 @@ the original author of the PR is asked if they will submit the relevant
+>   users can search for "all port PRs in a particular repository" for
+>   example.
+
+### Forward ports
+
+Forward ports tend to be much less common than backports. However,
+consolidating a number of standalone repositories into a single repository for
+the Kata 2.0 development effort introduced a potential forward port
+requirement. At the time, both Kata 1.x and 2.0 versions were being developed
+in parallel. This meant that lots of PRs raised in the Kata 1.x repositories
+needed to be forward ported to the Kata 2.0 repository (since this was to be
+the next major release and needed to contain all bug fixes and features where
+possible).
+
+| Initial PR raised | PR type | Backport? | Forward port? |
+|-|-|-|-|
+| Particular 1.x repository | bug fix | stable branches | 2.0 repository |
+| Particular 1.x repository | feature | | 2.0 repository |
+| The 2.0 repository | bug fix | 1.x (and maybe stable branches) | - |
+| The 2.0 repository | feature | | - |
+
+### Porting examples
+
+#### Backport and forward port example
+
+Imagine that you have just raised a new PR on a Kata 1.x repository. The PR
+contains three commits:
+
+- A commit to fix a "typo" in a comment.
+- A commit that changes the way a container is destroyed (and updates the tests).
+- A commit that updates the documentation explaining the new contain
+  destruction behaviour.
+
+Since the PR is not adding any new functionality and since it is correcting
+problems with existing code, it can be considered a bug fix PR. Bug fixes
+should generally be backported. However, this PR was raised in a Kata 1.x
+repository meaning it should also potentially be forward-ported to the Kata
+2.0 repository.
+
+Looking at the tables in the [porting labels](#porting-labels) and the [stable
+branch backports](#stable-branch-backports) sections shows that this PR needs
+to be labelled with the following labels:
+
+- `needs-backport`
+- `needs-forward-port`
+- `stable-candidate`
+
+Once this PR is approved and is merged in the Kata 1.x repository, it should
+now be backported and forward ported:
+
+- Backport: Raise new PRs on the currently maintained stable releases
+
+  - These PRs should be labelled with the `backport` label.
+  - The commit message should mention the original PR.
+  - The commit message should reference the *original* issue number in the
+    ["fixes" comment](#patch-format).
+
+- Forward port: Raise a new PR on the Kata 2.0 repository
+
+  - This PR should be labelled with the `forward-port` label.
+  - The commit message should mention the original PR.
+  - A new issue should be used for the PR and the original issue URL
+    referenced in the issue text.
+
+- Add [porting comments](#porting-comments) to the *original PR* with two comments,
+  one for each port:
+
+  ```
+  backport PR: https://github.com/kata-containers/runtime/pull/XXX
+  forward port PR: https://github.com/kata-containers/kata-containers/pull/YYY
+  ```
```

Comment:
W.r.t. the tooling, will it work only with "backport PR" and "forward port PR", and not something like "forward-port PR"?

Reply:
@cmaf - you are right, the tooling will actually recognise variants, but I didn't want to have to specify them here with a regex;)